json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
+        <td>No</td>
+        <td>A list of software components discovered by the <a href="#def-diagnostic"><strong>diagnostic</strong></a> which will be enumerated with <a href="#def-uniqueidentifier"><strong>unique IDs</strong></a> that can be used for later reference in <a href="#def-diagnostic"><strong>diagnostic</strong></a> output. This is an array of <a href="#softwareinfo"><em>SoftwareInfo</em></a> messages, of which the elements are detailed below.</td>
+    </tr>
+</table>
+
+
+##### Example
+
+```json
+{
+  "testRunArtifact": {
+    "testRunStart": {
+      "name": "mlc_test",
+      "version": "1.0",
+      "commandLine": "mlc/mlc --use_default_thresholds=true --data_collection_mode=true",
+      "parameters": {
+        "max_bandwidth": 7200.0,
+        "mode": "fast_mode",
+        "data_collection_mode": true,
+        "min_bandwidth": 700.0,
+        "use_default_thresholds": true
+      },
+      "dutInfo": {
+        "dutInfoId": "1",
+        "name": "ocp_lab_0222",
+        "platformInfos": [
+          {
+            "info": "memory_optimized"
+          }
+        ],
+        "softwareInfos": [
+          {
+            "softwareInfoId": "1",
+            "computerSystem": "primary_node",
+            "softwareType": "FIRMWARE",
+            "name": "bmc_firmware",
+            "version": "10",
+            "revision": "11"
+          }
+        ],
+        "hardwareInfos": [
+          {
+            "hardwareInfoId": "1",
+            "computerSystem": "primary_node",
+            "manager": "bmc0",
+            "name": "primary node",
+            "location": "MB/DIMM_A1",
+            "odataId": "/redfish/v1/Systems/System.Embedded.1/Memory/DIMMSLOTA1",
+            "partNumber": "P03052-091",
+            "serialNumber": "HMA2022029281901",
+            "manufacturer": "hynix",
+            "manufacturerPartNumber": "HMA84GR7AFR4N-VK",
+            "partType": "DIMM",
+            "version": "1",
+            "revision": "2"
+          }
+        ]
+      }
+    }
+  },
+  "sequenceNumber": 1,
+  "timestamp": "2022-07-25T04:28:12.903943Z"
+}
+```
+
+
+#### Error
+
+##### Description
+
+An [_error_](#error) is an output [**artifact**](#def-artifact) that reports a software, firmware, test or any other hardware-unrelated issues. [_Errors_](#error) can be child [**artifacts**](#def-artifact) of either [_TestRunArtifact_](#test-run-artifacts) or [_TestStepArtifact_](#test-step-artifacts) messages.
+
+##### Attributes
+
+<table>
+    <tr>
+        <td><strong>Attribute</strong></td>
+        <td><strong>Type</strong></td>
+        <td><strong>Required</strong></td>
+        <td><strong>Description</strong></td>
+    </tr>
+    <tr>
+        <td><em>symptom</em></td>
+        <td>string</td>
+        <td><strong>Yes</strong></td>
+        <td>A short string of the software issued <a href="#def-verdict"><strong>verdict</strong></a>.</td>
```

Comment:
Not sure that verdict is the right word for an error symptom.

Reply:
i remember we discussed this; was there another term to be used here? this was in the google doc latest